cmd/server.go

```diff
@@ -22,7 +22,7 @@ import (
 
 	"github.com/runatlantis/atlantis/server/logging"
 
-	"github.com/mitchellh/go-homedir"
+	homedir "github.com/mitchellh/go-homedir"
```

Comment:
My VSCode seems to not like that the package name doesn't line up with the path name. I can fix this if its an issue.

Reply:
Yes please fix.